lib/zache.rb

```diff
@@ -47,6 +47,11 @@ def exists?(key)
     @hash.key?(key)
   end
 
+  def remove(key)
```

Comment:
@NikitaNaumenko what about thread safety? We have to use `@mutex`, as it's done in `get`.

Reply:
@yegor256 i've done it, `remove()` should returns removed value or new hash without removed key?